mangaki/mangaki/utils/anidb.py

```diff
@@ -280,66 +283,60 @@ def _build_staff(self,
 
     def get_tags(self,
                  anidb_aid: Optional[int] = None,
-                 tags_soup: Optional[str] = None) -> Dict[str, Dict[str, Any]]:
+                 tags_soup: Optional[str] = None) -> List[AniDBTag]:
         """ Get the list of tags for an anime given the the AniDB ID or the XML
         string containing tags.
 
-        Return : Dict[tag_title: {'weight': int, 'anidb_tag_id': int}]
+        Return : List[AniDBTag]
         """
         if anidb_aid is not None:
             anime = self.get_xml(anidb_aid)
             tags_soup = anime.tags
 
-        tags = {}
-        if tags_soup is not None:
-            for tag_node in tags_soup.find_all('tag'):
-                tag_title = str(tag_node.find('name').string).strip()
-                tag_id = int(tag_node.get('id'))
-                tag_weight = int(tag_node.get('weight'))
-                tag_verified = tag_node.get('verified').lower() == 'true'
+        if tags_soup is None:
+            return []
+
+        tags = []
+        for tag_node in tags_soup.find_all('tag'):
+            tag_title = str(tag_node.find('name').string).strip()
+            tag_id = int(tag_node.get('id'))
+            tag_weight = int(tag_node.get('weight'))
+            tag_verified = tag_node.get('verified').lower() == 'true'
 
-                if tag_verified:
-                    tags[tag_title] = {'weight': tag_weight, 'anidb_tag_id': tag_id}
+            if tag_verified:
+                tags.append( AniDBTag(title=tag_title, weight=tag_weight, anidb_tag_id=tag_id) )
```

Comment:
If my tool is not wrong, it seems like you added a white space after the `append(` and before the final closing parenthesis, those are unusual in Python, could you remove them?
If you want to make it more readable, an alternative would be:
```python
tags.append(
    AniDBTag(
        title=tag_title,
        weight=tag_weight,
        anidb_tag_id=tag_id
    )
)
```